sormas-api/src/main/resources/captions.properties

```diff
@@ -61,6 +61,7 @@ assign=Assign
 assignToMe=Assign to me
 endOfProcessingDate=End of processing date
 dearchiveReason=De-archive reason
+deleteReason=Delete reason
```

Comment:
Reason for deletion (also change this in other places)

Reply:
changed